soda/core/soda/scan.py

```diff
@@ -370,14 +370,15 @@ def execute(self):
             fail_text = "failure" if checks_warn_count == 1 else "failures"
             error_count = len(self.get_error_logs())
             error_text = "error" if error_count == 1 else "errors"
-            self.__log_checks(None)
             checks_not_evaluated = len(self._checks) - checks_pass_count - checks_warn_count - checks_fail_count
 
+            if len(self._checks) == 0:
+                self._logs.warning("No checks found, 0 checks evaluated.")
```

Comment:
The remainder check logging should be put in an else branch of this `if len(self._checks) == 0:` because no other summary needs to be printed if there are no checks.

Reply:
indeed, we can save a bit of compute here. Changed, I left the cloud communication intact, so that we send empty scans as well, this might be useful info on user activity etc, would you agree?